### PR TITLE
[Storage][Blob][STG71]Managed Disk Page Range Diff 💿

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py [hunk 1]
```diff
@@ -2019,6 +2019,75 @@ def get_page_ranges( # type: ignore
             process_storage_error(error)
         return get_page_ranges_result(ranges)
 
+    @distributed_trace
+    def get_managed_disk_page_range_diff( # type: ignore
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py]: The decision (after the last discussion) is to add a parameter to the existing page diff API.

Comment (xiafu-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py]: It seems the other ask separate the disk diff method into a different one https://github.com/Azure/azure-sdk-for-net/pull/9195/files

Comment (zezha-msft) [reply sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py]: My bad, the decision was reverted. 🤣

File: sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py [hunk 2]
```diff
@@ -2019,6 +2019,75 @@ def get_page_ranges( # type: ignore
...
+            self, offset=None, # type: Optional[int]
+            length=None,  # type: Optional[int]
+            prev_snapshot_url=None,  # type: Optional[str]
+            **kwargs
+        ):
+        # type: (...) -> Tuple[List[Dict[str, int]], List[Dict[str, int]]]
+        """Returns the list of valid page ranges for a Page Blob or snapshot
+        of a page blob in managed disk account.
```
Comment (zezha-msft) [inline sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py]: managed disk account might be confusing, maybe consider:

Returns the list of valid page ranges for a managed disk or snapshot.

**Pull request merged**